roda-ui/roda-wui/src/main/resources/config/theme/theme.css

```
@@ -1,10 +1,900 @@
+  --eterna-slate: #4B5563;
+  --eterna-muted: #6B7280;
+  --eterna-brand-gray: #9C9E9F;
+  --eterna-border: #E5E7EB;
+  --eterna-surface: #F9FAFB;
+  --eterna-paper: #FFFFFF;
+  --eterna-dark: #111827;
+  --eterna-teal: #0E7490;
+  --eterna-ok: #059669;
+  --eterna-warn: #D97706;
+
+  --eterna-font-display: 'Barlow Condensed', 'Arial Narrow', 'Helvetica Neue', Arial, sans-serif;
```

Stylelint (17.9.0)

[error] Expected "Arial" to be "arial" (value-keyword-case)

(value-keyword-case)